app/models/app.rb

```diff
@@ -165,12 +165,16 @@ def copy_attributes_from(app_id)
     end
   end
 
-  def unresolved_count
-    @unresolved_count ||= problems.unresolved.count
+  def unresolved_count(query = {})
+    @unresolved_count ||= problems.unresolved.where(query).count
```

Comment:
Do note the caching in `@unresolved_count` does not take the `query` parameter into account, which means calling this method with different values for `query` will result in always returning the first computed response. Is this going to be a problem?

Reply:
Bump